banzai_nres/tests/test_rv.py

```diff
@@ -85,7 +85,7 @@ def load(self, *args):
     image.instrument = SimpleNamespace()
     image.instrument.site = 'npt'
     # Classification just can't be None so that the stage does not abort.
-    image.classification = 'foo'
+    image.classification = SimpleNamespace(**{'T_effective': 10.0, 'log_g': 0.0, 'metallicity': 0.0, 'alpha': 0.0})
```

Comment:
"10.0" isn't really a realistic Teff, I know this is just in the unit tests but it might be better to use a slightly more realistic value like "5000.0"

Reply:
Done.